internal/sweep/sweep_test.go

```diff
@@ -0,0 +1,27 @@
+package sweep_test
+
+import (
+	"testing"
+
+	"github.com/equinix/terraform-provider-equinix/equinix"
+	"github.com/equinix/terraform-provider-equinix/internal/resources/metal/project"
+	"github.com/equinix/terraform-provider-equinix/internal/resources/metal/ssh_key"
+	"github.com/equinix/terraform-provider-equinix/internal/resources/metal/vrf"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestMain(m *testing.M) {
+	// Register legacy sweepers for resources balled up under equinix/
```

Comment:
Tempting to stub out directories for these resources in `internal/...` and setup the sweepers as the only residents ahead of further migration. It would make those migrations smaller in scope and keep this PR scoped to sweepers without the `equinix.` hack.

I'm surprised we don't have Sweepers for non-Metal resources.

Reply:
Yeah, I like the idea of just moving these sweepers to internal now.  This went through a few iterations and it's no longer necessary to keep the Metal sweepers in `equinix/` so keeping them there is just adding churn.  We'll still need the Makefile change because non-Metal sweepers are still under Equinix (it's fine to have them defined across multiple packages in general; the reason the Metal ones all have to be added in the same TestMain is because they have interdependencies).